lua/actions-preview/backend/snacks.lua

```diff
@@ -1,5 +1,25 @@
 local M = {}
 
+--- function called to format item to list entry
+---@param item snacks.picker.finder.Item
+local function format(item)
+  local ret = {} ---@type snacks.picker.Highlight[]
+
+  local idx = tostring(item.idx)
+  idx = (" "):rep(#tostring(item._count) - #idx) .. idx
+
+  ret[#ret + 1] = { idx .. ".", "SnacksPickerIdx" }
```

Comment:
Since this code is meant to adjust the width, I think using `format` would make it more readable.
```suggestion
  local idx = ("%%%ds."):format(#tostring(item._count)):format(item.idx)
  ret[#ret + 1] = { idx, "SnacksPickerIdx" }
```